types/index.d.ts

```diff
@@ -1828,6 +1828,8 @@ declare namespace Knex {
     currentVersion(config?: MigratorConfig): Promise<string>;
     up(config?: MigratorConfig): Promise<any>;
     down(config?: MigratorConfig): Promise<any>;
+    list(config?: MigratorConfig): Promise<any>;
```

Comment:
I think list is already included and can be removed from this PR?

Reply:
Where?

Reply:
https://github.com/knex/knex/blob/master/types/index.d.ts#L1830

Reply:
Ah, ok, thanks. :)